src/commands/status/status.ts

```diff
@@ -42,16 +42,8 @@ export const status = async (options: OptionValues, command: BaseCommand) => {
     Name: user.full_name,
     Email: user.email,
     GitHub: ghuser,
+    Teams: accounts.map(({ name }) => name),
   }
-  const teamsData = {}
-
-  accounts.forEach((team) => {
-    // @ts-expect-error TS(7053) FIXME: Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
-    teamsData[team.name] = team.roles_allowed.join(' ')
-  })
-
-  // @ts-expect-error TS(2339) FIXME: Property 'Teams' does not exist on type '{ Name: a... Remove this comment to see the full error message
-  accountData.Teams = teamsData
```

Comment:
Technically this makes this PR breaking, since we're changing the output format and contents (including the structured json output). Was that intentional? If so, we should update the PR title to cut a major, and mention the output change in the PR description.

Reply:
It's a porcelain interface, so this is not a breaking change. (We noted that this would be a breaking change in the issue for this, and that the `--json` output for this command isn't working--for me, anyway.)

Reply:
This changes both the porcelain output and the `--json` output, both of which work for me at the moment. 🤔

Reply:
Noted! I'll make this a breaking change, then.